templates/includes/navbar.html

```diff
@@ -129,6 +129,9 @@
                     <a href="/django" style="color: #db1212; letter-spacing: 1px;" class="nav-item nav-link">Django</a>
                 </div>
             </div>
+            <a href="{% url 'home-team' %}" style="font-size:20px; color: #1504fa; letter-spacing: 1px; font-weight: bold;" class="nav-item nav-link">
+                Our Team
+            </a>
```

Comment:
This anchor tag is causing the error. There is no URL mapped with the name 'home-team'. Kindly map it to the correct url.

Reply:
actually his branch is 54 commits behind 
this page is added recently help him to update his branch otherwise it will effects other pages